com.unity.cinemachine/Runtime/Core/InputAxisControllerBase.cs

```diff
@@ -30,20 +30,6 @@ public interface IInputAxisController
     [SaveDuringPlay]
     public abstract class InputAxisControllerBase<T> : MonoBehaviour, IInputAxisController where T : IInputAxisReader, new ()
     {
-        /// <summary>
-        /// Leave this at -1 for single-player games.
-        /// For multi-player games, set this to be the player index, and the actions will
-        /// be read from that player's controls
-        /// </summary>
-        [Tooltip("Leave this at -1 for single-player games.  "
-            + "For multi-player games, set this to be the player index, and the actions will "
-            + "be read from that player's controls")]
-        public int PlayerIndex = -1;
-
-        /// <summary>If set, Input Actions will be auto-enabled at start</summary>
-        [Tooltip("If set, Input Actions will be auto-enabled at start")]
-        public bool AutoEnableInputs = true;
-        
```

Comment:
You are right to remove this from the base class, as it's only relevant to our default implementation